content/docs/reference-react.md

```diff
@@ -124,7 +124,7 @@ const MyComponent = React.memo(function MyComponent(props) {
 });
 ```
 
-`React.memo` is a [higher order component](/docs/higher-order-components.html). It's similar to [`React.PureComponent`](#reactpurecomponent) but for function components instead of classes.
+`React.memo` is a [higher order component](/docs/higher-order-components.html). It's similar to [`React.PureComponent`](#reactpurecomponent). It can be used with both functional components and classes.
```

Comment:
Let's just drop this paragraph altogether?

Reply:
ok

Reply:
@gaearon does this look ok now?